model/error/event_test.go

```diff
@@ -188,17 +189,18 @@ func TestErrorEventDecode(t *testing.T) {
 			},
 		},
 	} {
-		for _, decodeFct := range []func(interface{}, error) (transform.Transformable, error){V1DecodeEvent, V2DecodeEvent} {
+		for idx, decodeFct := range []func(interface{}, error) (transform.Transformable, error){V1DecodeEvent, V2DecodeEvent} {
 			transformable, err := decodeFct(test.input, test.inpErr)
 
 			if test.e != nil {
+				fmt.Println(idx)
```

Comment:
did you mean to leave that Println in?

Reply:
Thanks for catching this, will remove in a follow up.